components/PageHeader.vue

```diff
@@ -15,39 +15,29 @@
       >
     </b-navbar-brand>
     <b-navbar-nav class="ml-auto w-100 col-md-5 col-lg-4 p-0">
-      <b-nav-form
-        class="justify-content-center justify-content-sm-end"
-        @submit.prevent="submitSearchForm"
-      >
-        <b-form-input
-          v-model="query"
-          size="sm"
-          type="text"
-          class="mr-2 w-75"
-          placeholder="What are you looking for?"
-          name="query"
-          data-qa="search box"
-        />
-        <b-button
-          size="sm"
-          type="submit"
-          data-qa="search button"
-        >
-          <img
-            src="../assets/img/magnifier.svg"
-            alt="Search"
-          >
-        </b-button>
-      </b-nav-form>
+      <SearchForm
+        v-model="query"
+        :is-loading="isLoading"
+        :in-header="inHeader"
+        data-qa="search form header"
```

Comment:
'header' is redundant, we can infer this in both unit and e2e tests by selecting the b-navbar first element which has a 'data-qa="header"' already.

Reply:
in fact 'data-qa="search form"' is applied to the form itself so we don't need any qa attribute here at all.

Reply:
I have mounted the component, so I am able to check if the form exists. I left the data-qa="search form" on the form in the actual component.